hypervisor/mmu.c

```diff
@@ -0,0 +1,31 @@
+#include "common.h"
+#define MAX_PAGE_PDE    8
+unsigned int page_directory[1024] __attribute__((aligned(4096)));
+unsigned int page_tables[MAX_PAGE_PDE][1024] __attribute__((aligned(4096)));
+
+
+void init_page_directory() {
```

Comment:
this function obviously not only init page directory, but also init page entry. so i suggest change name to init_page or page_setup.

Reply:
Actually, the operations to page_tables, just only for init page_dirctory.not for others.

Reply:
so what's the action of line31??